packages/core/src/waitFor.ts

```diff
@@ -0,0 +1,51 @@
+import { ActorRef, EmittedFrom } from '.';
+
+interface WaitForOptions<_TActorRef extends ActorRef<any, any>> {
```

Comment:
What do we expect to add here if we are "reserving" a generic slot for it?

Reply:
Nothing right now... I'll remove it since it's only to be used internally.